### PR TITLE
Add script to index ES transferfile entries in the SS

### PR DESCRIPTION
**Pull request by mistydemeo**

### CODE REVIEW

File: tools/reindex-backlogged-transfers [hunk 1]
```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/python
```
Comment (Hwesta) [inline tools/reindex-backlogged-transfers]: Works with Python 2 or 3? :D

**Comment by Hwesta** [comment]
:+1:

File: tools/reindex-backlogged-transfers (hunk 1 above)
Comment (mistydemeo) [inline tools/reindex-backlogged-transfers]: Haha, I haven't tested. I should...

Comment (mistydemeo) [inline tools/reindex-backlogged-transfers]: Apparently *slumber* doesn't work in Python 3 😰

**Pull request merged**